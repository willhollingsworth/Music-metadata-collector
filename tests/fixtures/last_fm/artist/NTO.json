{
  "artist": {
    "name": "NTO",
    "url": "https://www.last.fm/music/NTO",
    "image": [
      {
        "#text": "https://lastfm.freetls.fastly.net/i/u/34s/2a96cbd8b46e442fc41c2b86b821562f.png",
        "size": "small"
      },
      {
        "#text": "https://lastfm.freetls.fastly.net/i/u/64s/2a96cbd8b46e442fc41c2b86b821562f.png",
        "size": "medium"
      },
      {
        "#text": "https://lastfm.freetls.fastly.net/i/u/174s/2a96cbd8b46e442fc41c2b86b821562f.png",
        "size": "large"
      },
      {
        "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
        "size": "extralarge"
      },
      {
        "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
        "size": "mega"
      },
      {
        "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
        "size": ""
      }
    ],
    "streamable": "0",
    "ontour": "1",
    "stats": {
      "listeners": "186855",
      "playcount": "2842994"
    },
    "similar": {
      "artist": [
        {
          "name": "Joachim Pastor",
          "url": "https://www.last.fm/music/Joachim+Pastor",
          "image": [
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/34s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "small"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/64s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "medium"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/174s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "large"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "extralarge"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "mega"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": ""
            }
          ]
        },
        {
          "name": "Worakls",
          "url": "https://www.last.fm/music/Worakls",
          "image": [
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/34s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "small"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/64s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "medium"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/174s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "large"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "extralarge"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "mega"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": ""
            }
          ]
        },
        {
          "name": "Joris Delacroix",
          "url": "https://www.last.fm/music/Joris+Delacroix",
          "image": [
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/34s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "small"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/64s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "medium"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/174s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "large"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "extralarge"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "mega"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": ""
            }
          ]
        },
        {
          "name": "Teho",
          "url": "https://www.last.fm/music/Teho",
          "image": [
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/34s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "small"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/64s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "medium"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/174s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "large"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "extralarge"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "mega"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": ""
            }
          ]
        },
        {
          "name": "Thylacine",
          "url": "https://www.last.fm/music/Thylacine",
          "image": [
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/34s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "small"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/64s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "medium"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/174s/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "large"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "extralarge"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": "mega"
            },
            {
              "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png",
              "size": ""
            }
          ]
        }
      ]
    },
    "tags": {
      "tag": [
        {
          "name": "minimal",
          "url": "https://www.last.fm/tag/minimal"
        },
        {
          "name": "electronic",
          "url": "https://www.last.fm/tag/electronic"
        },
        {
          "name": "french",
          "url": "https://www.last.fm/tag/french"
        },
        {
          "name": "techno",
          "url": "https://www.last.fm/tag/techno"
        },
        {
          "name": "france",
          "url": "https://www.last.fm/tag/france"
        }
      ]
    },
    "bio": {
      "links": {
        "link": {
          "#text": "",
          "rel": "original",
          "href": "https://last.fm/music/NTO/+wiki"
        }
      },
      "published": "13 Sep 2020, 22:33",
      "summary": "The story behind NTO started with a match made in heaven, a kid from the South of France always hooked to his guitar, and electronic music. After going to the Sziget Festival when he was 18, NTO started to hang out regularly at La Villa Rouge, and Bar Live, two legendary clubs in the South of France. At the time, German minimal techno music was booming and Anthony Favier would spend his nights listening to Stephan Bodzin, James Holden, or the very beginnings of Paul Kalkbrenner. <a href=\"https://www.last.fm/music/NTO\">Read more on Last.fm</a>",
      "content": "The story behind NTO started with a match made in heaven, a kid from the South of France always hooked to his guitar, and electronic music. After going to the Sziget Festival when he was 18, NTO started to hang out regularly at La Villa Rouge, and Bar Live, two legendary clubs in the South of France. At the time, German minimal techno music was booming and Anthony Favier would spend his nights listening to Stephan Bodzin, James Holden, or the very beginnings of Paul Kalkbrenner.\n\nThe story behind NTO is also about his friendship with Worakls and Joachim Pastor, with whom he created Hungry Music in 2013. Performing together on the Olympia\u2019s stage to the Vieilles Charrues Festival, they learned to master the stage. Thanks to his hits \"Trauma\" and \"La cl\u00e9 des champs\", which account for several millions of streams on DSPs, NTO's career took off internationally.\n\nToday, NTO is about to start a new chapter of his career. His first single \u201cThe Kid and the Planes\u201d is now available. \n\nAKA n'to. <a href=\"https://www.last.fm/music/NTO\">Read more on Last.fm</a>. User-contributed text is available under the Creative Commons By-SA License; additional terms may apply."
    }
  }
}