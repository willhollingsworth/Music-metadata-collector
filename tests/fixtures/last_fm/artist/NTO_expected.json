{
  "artist_name": "NTO",
  "artist_url": "https://www.last.fm/music/NTO",
  "listeners": "187066",
  "playcount": "2847934"
}